alerts/cluster-monitoring-operator/PrometheusRemoteStorageFailures.md

```diff
@@ -0,0 +1,62 @@
+# PrometheusRemoteStorageFailures
+
+## Meaning
+
+The `PrometheusRemoteStorageFailures` alert triggers when there has been
+a constant increase in failures sending samples to the remote storage for more
+than 15 minutes, in either the platform Prometheus or a user-defined monitoring
+Prometheus pods..
+
+## Impact
+
+Prometheus samples in remote storage might be missing. Depending on the
+remote_write pipeline configuration, prometheus memory usage might increase
+while pending samples are queued.
+
+## Diagnosis
+
+* Determine if the alert has been triggered for the instance of Prometheus
+  instance used for default cluster monitoring or for the instance that
+  monitors user-defined projects by checking the alert message's `namespace`
+  label. The namespace value will indicate the Prometheus instance.
+
+  E.g. `openshift-monitoring` for default platform monitoring and
+  `openshift-user-workload-monitoring` for user-defined monitoring.
+
+* Review the logs for the affected Prometheus instance:
+
+  ```console
+  $ NAMESPACE='<value of namespace label from alert>'
+
+  $ oc -n $NAMESPACE logs -l 'app.kubernetes.io/component=prometheus'
+  level=error ... msg="Failed to send batch, retrying" ...
+  ```
+
+* Check prometheus logs and remote storage logs.
+
+## Mitigation
+
+This alert indicates an issue in Prometheus' communication with the remote
+system. The cause can be on either the Prometheus side or the remote side.
+Common issues are:
+
+* Failure to authenticate to the remote storage system
+  * Check the authentication parameters in the CMO and user-workload
+      configmaps.
```

Comment:
```suggestion
```

Reply:
will drop this one as per the change on the above suggestion.